src/telegraf.ts

```diff
@@ -69,30 +59,21 @@ namespace Telegraf {
   }
 }
 
-const allowedUpdates: tt.UpdateType[] | undefined = undefined
-
 export class Telegraf<
   TContext extends Context = Context
```

Comment:
As you're touching this file anyway, it might be the right time to rename `TContext` to `C` here, as well.